File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_03_MoreExercises/_04_RefactoringPrimeChecker.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._03_MoreExercises;

import java.util.Scanner;

public class _04_RefactoringPrimeChecker {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Check if the number is prime or not
        for (int i = 2; i <= n; i++) {
            boolean isPrime = true;

            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    isPrime = false;
                    break;
                }
            }

            // Print result
            System.out.printf("%d -> %b%n", i, isPrime);
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_03_MoreExercises/_04_RefactoringPrimeChecker/_04_RefactoringPrimeChecker.cs
using System;

public class _04_RefactoringPrimeChecker
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Check if the number is prime or not
        for (int i = 2; i <= n; i++)
        {
            bool isPrime = true;

            for (int j = 2; j < i; j++)
            {
                if (i % j == 0)
                {
                    isPrime = false;
                    break;
                }
            }

            // Print result
            Console.WriteLine($"{i} -> {isPrime.ToString().ToLower()}");
        }
    }
}
